Automattic/wp-calypso
# Listing content on /advertising by number of views - POC 2

Related to 1693-gh-Automattic/dotcom-forge

## Proposed Changes

Here we're showing /advertising posts/pages by their views numbers

## Testing Instructions



* Apply the Diff in your sandbox: https://github.com/Automattic/jetpack/pull/29095
* Access a popular site you have and make sure we're listing posts and pages by their views at the top of /advertising items
* Tip: Comment [these lines](https://github.com/Automattic/wp-calypso/blob/update/filter-advertising-by-views-v2/client/my-sites/promote-post/main.tsx#L122-L143) and access any private site from automattic that have more views, like the explorers p2.

## Pre-merge Checklist
